source/android/adaptivecards/src/AdaptiveCardObjectModel.i

```diff
@@ -101,6 +101,8 @@ struct tm {
 #include "../../../shared/cpp/ObjectModel/Media.h"
 #include "../../../shared/cpp/ObjectModel/ToggleVisibilityAction.h"
 #include "../../../shared/cpp/ObjectModel/ToggleVisibilityTarget.h"
+%include "../../../shared/cpp/ObjectModel/UnknownElement.h"
+%include "../../../shared/cpp/ObjectModel/UnknownActionElement.h"
```

Comment:
these should go in the list of `%include` directives below

Reply:
thanks

---
In reply to: [251642200](https://github.com/Microsoft/AdaptiveCards/pull/2325#discussion_r251642200) [](ancestors = 251642200)